solr/solrj/src/java/org/apache/solr/common/cloud/DocCollection.java

```diff
@@ -385,6 +394,15 @@ public boolean isReadOnly() {
     return readOnly;
   }
 
+  /**
+   * When this DocCollection is read from Zookeeper, it captures a collection persisted in
+   * Zookeeper. The date thus captures the date when the collection node was created, and thus the
+   * collection made usable by APIs as specified by {@code ZkStateReader#getCollectionPath(String)}
```

Comment:
Should start with a simple sentence like "The creation time of this collection."  Perhaps say it's never null. *Then* the proposed text.